types/decode/layout.d.ts

```diff
@@ -16,6 +16,7 @@ export interface DomData {
     parent: number;
     next: number;
     tag: string;
+    position: number;
```

Comment:
>number [](start = 14, length = 6)

Very minor but could we declare a type alias to distinguish integer number versus floating point? It makes it easier to know for a human reader. It will have no impacting on typing though.

Reply:
I could but in clarity anytime we use a number it's always an integer. We don't capture floats unless you believe I am missing something here.